app/assets/javascripts/discourse/widgets/post-cooked.js

```diff
@@ -52,13 +56,13 @@ export default class PostCooked {
 
     if (highlight && highlight.length > 2) {
       if (this._highlighted) {
-        $html.unhighlight();
+        unhighlightHTML($html[0]);
```

Comment:
probably worth having a variable with a good name, instead of reach for $html[0] each time